scipy/io/harwell_boeing/tests/test_hb.py

```diff
@@ -58,6 +57,20 @@ def test_simple(self):
         assert_csc_almost_equal(m, SIMPLE_MATRIX)
 
 
+class TestHBReadWrite(object):
```

Comment:
Seems like this could be added to `TestRBRoundtrip`, instead of adding a new test class.

Reply:
I removed `TestRBRoundtrip` and left over only `TestHBReadWrite` because it is a more descriptive name.